### ISSUE TITLE
GMN application

### ISSUE THREAD

**Issue by simba0626** [deepmind/deepmind-research#58]
Hi author, 
The work is very well.   I want to use this work for question answering. 
 I have one question, when run codes:  I use 200-d vector for representiong node and edge features. After running, I find the attention weights can not align nodes well when compute two graphs match. Like this:

![图片1](https://user-images.githubusercontent.com/8748445/84756812-de523800-aff5-11ea-9378-54e9172005a5.png)

I do not know where is it wrong. Is it features vector error or others ?

Thank you

ywsun

**Comment by yujiali** [comment]
Hi ywsun,

It may help if you play with the temperature of the attention logits a bit to get a more visible attention pattern.  For example you could multiply the attention score s(h_i, h_j) with a factor of 1/T, and then change T to something small, potentially that will give you a clearer pattern.

**Comment by joehoeller** [comment]
@yujiali 
What is the shape of the inputs I can generate for training?
For example, I have seen some tools take a tuple [ (0,1), (1,2), (2,1)...] where node 0 connects to node 1, and node 1 connects to node 2. I have also seen JSON representations of graphs as well as TSV files where their values make a graph. 

I simply want to generate whatever data and shape is needed to train the model to test it out. I hope this makes sense.

**Comment by RZZBlackMagic** [comment]
@yujiali @simba0626 hello, I would like to learn Graph Matching Networks, Is there a public data set for this project? IF not, Could  you tell me the format of the dataset?

Thanks
